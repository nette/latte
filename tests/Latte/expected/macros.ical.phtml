<?php
// source: %A%

class Template%a% extends Latte\Template
{

	function render()
	{
%A%
header('Content-Type: text/calendar; charset=utf-8') ;$start = '2011-06-06'; $end = '2011-06-07'; $info = "Hello \"hello\", \nWorld" ?>
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//example.org//iCal 4.0.3//CS
METHOD:PUBLISH
BEGIN:VEVENT
DTSTAMP:<?php echo LFilters::escapeIcal(call_user_func($this->filters->date, $start, 'Ymd\THis')) ?>

DTSTART;TZID=Europe/Prague:<?php echo LFilters::escapeIcal(call_user_func($this->filters->date, $start, 'Ymd\THis')) ?>

DTEND;TZID=Europe/Prague:<?php echo LFilters::escapeIcal(call_user_func($this->filters->date, $end, 'Ymd\THis')) ?>

SUMMARY;LANGUAGE=cs:<?php echo LFilters::escapeIcal($info) ?>

DESCRIPTION:
CLASS:PUBLIC
END:VEVENT
END:VCALENDAR
<?php
	}

}
