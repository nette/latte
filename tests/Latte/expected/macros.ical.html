BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//example.org//iCal 4.0.3//CS
METHOD:PUBLISH
BEGIN:VEVENT
DTSTAMP:20110606T000000
DTSTART;TZID=Europe/Prague:20110606T000000
DTEND;TZID=Europe/Prague:20110607T000000
SUMMARY;LANGUAGE=cs:Hello \"hello\"\, \nWorld
DESCRIPTION:
CLASS:PUBLIC
END:VEVENT
END:VCALENDAR
